crates/ty_python_semantic/resources/mdtest/attributes.md

```diff
@@ -1759,6 +1767,7 @@ def external_getattribute(name) -> int:
 
 class ThisFails:
     def __init__(self):
+        # error: [invalid-assignment] "Implicit shadowing of function `__getattribute__`"
```

Comment:
There are a few new "implicit shadowing of function …" diagnostics in the [diff](https://shark.fish/type-of-self-diff.html). This seems sort-of fine, but maybe worth looking into.